### ISSUE TITLE
Lore not displayed

### ISSUE THREAD

**Issue by SlimeDog** [RockinChaos/ItemJoin#189]
Spigot 1.13.2 latest
ItemJoin 4.9.1-b155

Custom lore is not displayed with the item in slot 17. Custom lore is displayed if I change the slot to the hotbar (say, slot 7).

```
  identifier-lobby:
    id: BRICKS
    slot: 17
    count: 1
    name: '&bLobby'
    lore:
      - 'world identifier'
      - 'op only'
    permission-node: 'blackdog.op'
    itemflags: cancel-events, death-drops, hide-attributes, hide-durability, inventory-modify, placement, self-drops, unbreakable
    triggers: join, respawn, world-change
    enabled-worlds: world_lobby
```
No errors on **/itemjoin reload**.

**Comment by RockinChaos** [comment]
I apologize for the late reply, bouncing back and forth between the new and older framework of ItemJoin is a bit jarring.

I have tested this but I am unable to replicate this issue.
4.9.1-b155 (tested and shows the custom lore).
I have attached a screenshot, please let me know if I am missing something.

![tested](https://user-images.githubusercontent.com/10971577/49848158-9ca15b00-fd88-11e8-982a-d779e938c79d.png)

**Comment by SlimeDog** [comment]
Well, sure. Current n(plugins_=86. But none of them advertise inventory sterilization. Which plugins were you thinking of? I'll send a list in PM.
![image](https://user-images.githubusercontent.com/17748923/49849219-631f1e80-fd8d-11e8-8b1b-d1780fd1e1ef.png)

**Comment by SlimeDog** [comment]
Interestingly, lore for the CLOCK in slot 8, which is also an ItemJoin item, is visible.

**Comment by SlimeDog** [comment]
More experimentation. I changed the BRICKS (above) to CLOCK. The lore is visible. So type of item is important. I suspect it has to do with whether lore can be assigned "naturally" to the item.

**Comment by SlimeDog** [comment]
More experimentation. I changed the CLOCK back to BRICKS. Reloaded ItemJoin, and /back and /back to reload the IJ items. The lore is visible. WTF?

**Comment by RockinChaos** [comment]
Like I said its sounding more and more like a plugin sterilized the item randomly. Also if you were messing with animation itemflag on the bricks item before the b145 I think, there was a bug fix for the lore not being displayed properly on animated items so it would set a blank lore.

**Comment by SlimeDog** [comment]
I don't use the animate itemflag. The lore specification exists in the items.yml file (per above). It just isn't (wasn't) displayed.

Should I just reset all the items that are currently missing lore, and call it "fixed"?

**Comment by RockinChaos** [comment]
I would say that is the best idea but if it reappears as an issue it is definitely another plugin. I will continue to go down the whole list of plugins you sent me though and see if I find anything.

**Comment by SlimeDog** [comment]
Thanks.

**Comment by SlimeDog** [comment]
The newly-displayed lore is retained across server restarts. That's good. :)

**Comment by SlimeDog** [comment]
You know, I test a lot of plugins. It could be one that I tested and discarded as not very good.

**Comment by RockinChaos** [comment]
Hmmm possibly true. I am 31 plugins down so far lol.

**Comment by SlimeDog** [comment]
Since the lore is defined by ItemJoin, is it possible to enforce it when the objects are loaded? I imagine that I could construct a test case, by removing the lore, then replacing the definitions before a reload.

**Comment by RockinChaos** [comment]
This issue (if it is specific to ItemJoin would have been most likely officially fixed in the release of ItemJoin v5.0.0.
You can grab it here; https://www.spigotmc.org/resources/itemjoin.12661/download?version=254085

Please read the changelog as the documentation has not been updated yet;
https://www.spigotmc.org/wiki/recent-changes

(I apologize for the late response, I was unable to completely go through the plugin list you sent me due to lack of time while preparing for this update). Please re-open this issue if the same issue occurs.
